taxi/views.py

```diff
@@ -52,3 +54,43 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarCreateView(generic.CreateView, LoginRequiredMixin):
```

Comment:
Mixins are usually placed in a first position, consider it in your future classes.